src/components/StatePicker/StateSelectorModal.tsx

```diff
@@ -88,13 +82,14 @@ function StateSelectorModal({currentState, isVisible, onClose, onStateSelected,
                 testID={StateSelectorModal.displayName}
             >
                 <HeaderWithBackButton
-                    title={label || translate('common.state')}
+                    title={label ?? translate('common.state')}
```

Comment:
if label is an empty string '' then this will return '', could you confirm if that case will cause a problem here?

Reply:
I agree it will be more safe to use logical OR in this case, so [updated](https://github.com/Expensify/App/pull/33352/commits/089c626465bbc3a3280774717bf8b97f803b3679) it